.github/workflows/ci-release.yml

```diff
@@ -144,7 +144,7 @@ jobs:
       - name: Add Problem Matcher
         run: echo "::add-matcher::.github/matchers/tap.json"
       - name: Test
-        run: npm test --ignore-scripts -ws -iwr --if-present
+        run: npm test --ignore-scripts -iwr --if-present
```

Comment:
`-iwr` doesn't make sense outside of the `-ws` context.

Reply:
Good call, it doesn't break old npm versions like `-ws` does, but no reason to keep it in these cases.